lib/archethic/beacon_chain/summary_timer.ex

```diff
@@ -98,6 +99,24 @@ defmodule ArchEthic.BeaconChain.SummaryTimer do
     end
   end
 
+  def handle_info(
+        :next_summary_time,
+        state = %{
+          interval: interval
+        }
+      ) do
+    timer = schedule_next_summary_time(interval)
+
+    slot_time = DateTime.utc_now() |> Utils.truncate_datetime()
+
+    PubSub.notify_next_summary_time(next_summary(slot_time))
```

Comment:
but I wanted to know will it make any difference?

Reply:
Yes in terms of performance, as it will require `ets` lookup + CronTab next scheduling time. Also,  the computation is not necessary, as you are on summary time.